packages/fast-components-react-msft/src/carousel/carousel.props.ts

```diff
@@ -1,25 +1,84 @@
-import React from "react";
+import React, { ReactNodeArray } from "react";
```

Comment:
Doesn't look like you are using `ReactNodeArray` anywhere

Reply:
Yeah, I recall an errant tab where this got auto added. Removing.